### ISSUE TITLE
Person image upload feature not showing files and media like chrome

### ISSUE THREAD

**Issue by dshokouhi** [home-assistant/android#959]
**Home Assistant Android version:**
Latest master
**Android version:**
11
**Phone model:**
Pixel 4 XL
**Home Assistant version:**
0.115.1
**Last working Home Assistant release (if known):**
N/A new feature
**Description of problem:**

The new person image upload feature does not prompt the user for permissions nor does it show the file dialog to let them select an image.  In chrome this works as expected, in our app we don't get storage permissions yet so we may need to add permissions or also handle showing the prompt?

**Traceback (if applicable):**

```
N/A nothing showed up in logcat while reproducing the issue
```

**Screenshot of problem:**

![image](https://user-images.githubusercontent.com/1634145/93686836-f1016080-fa6d-11ea-9ccc-3e9e6854c718.png)

**Additional information:**

**Comment by dshokouhi** [comment]
Simply adding the permissions to the app was not good enough it seems 🤔

**Comment by dshokouhi** [comment]
Fixed in #986
